robocorp-code/vscode-client/src/activities.ts

```diff
@@ -156,19 +156,28 @@ export async function resolveInterpreter(targetRobot: string): Promise<ActionRes
             "robot.resolveInterpreter",
             targetRobot
         );
+        if (interpreter === null || (typeof interpreter === "string" && interpreter === "null")) {
+            throw Error("Interpreter not found. Retrying call...");
+        }
+        vscode.window.showInformationMessage(`Environment built & cached. Python interpreter loaded.`);
         return { "success": true, "message": "", "result": interpreter };
     } catch (error) {
         // We couldn't resolve with the robotframework language server command, fallback to the robocorp code command.
         try {
-            let result: ActionResult<InterpreterInfo | undefined> = await commands.executeCommand(
+            let interpreter: ActionResult<InterpreterInfo | undefined> = await commands.executeCommand(
                 roboCommands.ROBOCORP_RESOLVE_INTERPRETER,
                 {
                     "target_robot": targetRobot,
                 }
             );
-            return result;
+            if (interpreter === null || (typeof interpreter === "string" && interpreter === "null")) {
+                throw Error("Interpreter not found");
+            }
+            vscode.window.showInformationMessage(`Environment built & cached. Python interpreter loaded.`);
+            return interpreter;
         } catch (error) {
             logError("Error resolving interpreter.", error, "ACT_RESOLVE_INTERPRETER");
+            vscode.window.showErrorMessage(`Error resolving interpreter: ${error}`);
```

Comment:
At this point, as this can be called many times from different places, I think we shouldn't show an error here either (may be wanted in ROBOCORP_PACKAGE_ENVIRONMENT_REBUILD though).